README.md

```diff
@@ -111,7 +111,7 @@ If you want to change the directory, you have two options.
 First options is to set the full namespace while you're creating the presenter class
 
 ```bash
-php artisna presneter:make App\Models\Presenter\UserPresenter
+php artisan presneter:make App\Models\Presenter\UserPresenter
```

Comment:
Maybe you want to create an alias for the command? Something like `php artisan make:presenter`, without removing the current?

Reply:
This was just a spelling error change.